packages/wdio-applitools-service/package.json

```diff
@@ -34,7 +34,7 @@
     "url": "https://github.com/webdriverio/webdriverio/issues"
   },
   "dependencies": {
-    "@wdio/eyes.webdriverio": "^2.0.2",
+    "@applitools/eyes-webdriverio": "^5.1.0-beta.1",
```

Comment:
is it ok that version is beta?

Reply:
yeah, they said they are working on some missing things but the main functionality is there. I tested it and it works already better than my fork.